cmds/kcl-go/command/cmd_list_app.go

```diff
@@ -115,6 +121,40 @@ func NewLispAppCmd() *cli.Command {
 				}
 			}
 
+			if flagUseFastParser {
```

Comment:
add more comments on the `use-fast-parser` flag.

Reply:
update describe for the flag